recipes/LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2019 Phil Budne
```

Comment:
Where does this license file come from?

Reply:
the up stream developer didn't provide the license file in the pypi package. It just stated its MIT. So I created one according to the [guide](https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-licence-manually). Is that an appropriate way?

Reply:
The licence text is included in the main source file as:

```
#	Copyright (c) 2007 Stefan Goessner (goessner.net)
#       Copyright (c) 2008 Kate Rhodes (masukomi.org)
#       Copyright (c) 2008-2018 Philip Budne (ultimate.com)
#	Licensed under the MIT licence:
#
#	Permission is hereby granted, free of charge, to any person
#	obtaining a copy of this software and associated documentation
#	files (the "Software"), to deal in the Software without
#	restriction, including without limitation the rights to use,
#	copy, modify, merge, publish, distribute, sublicense, and/or sell
#	copies of the Software, and to permit persons to whom the
#	Software is furnished to do so, subject to the following
#	conditions:
#
#	The above copyright notice and this permission notice shall be
#	included in all copies or substantial portions of the Software.
#
#	THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
#	EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
#	OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
#	NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
#	HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
#	WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
#	FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
#	OTHER DEALINGS IN THE SOFTWARE.
```